docs/DISCOVERABLE_PARTITIONS.md

```diff
@@ -0,0 +1,354 @@
+memory via <tt>tmpfs</tt> and does not require a partition on disk. In some
+cases it might be desirable to make <tt>/tmp/</tt> persistent too, in which
+case it is recommended to make it a symlink or bind mount to
+<tt>/var/tmp/</tt>, thus not requiring its own partition type UUID.</td>
+</tr>
+
+<tr>
+<td><nobr><tt>0657fd6d-a4ab-43c4-84e5-0933c84b4f4f</tt></nobr></td>
+<td><nobr><i>Swap</i></nobr></td>
+<td>Swap</td>
+<td>All swap partitions on the disk containing the root partition are automatically enabled.</td>
+</tr>
+
+<tr>
+<td><nobr><tt>c12a7328-f81f-11d2-ba4b-00a0c93ec93b</tt></nobr></td>
+<td><nobr><i>EFI System Partition</i></nobr></td>
+<td>VFAT</td>
+<td>The ESP used for the current boot is automatically mounted to
+<tt>/efi/</tt> (or <tt>/boot/</tt> as fallback), unless a different partition
+is mounted there (possibly via <tt>/etc/fstab</tt>, or because the Extended
+Boot Loader Partition — see below — exists) or the directory is non-empty on
+the root disk.  This partition type is defined by the <a
+href="http://www.uefi.org/specifications">UEFI Specification</a>.</td>
+</tr>
+
+<tr>
+<td><nobr><tt>bc13c2ff-59e6-4262-a352-b275fd6f7172</tt></nobr></td>
+<td><nobr><i>Extended Boot Loader Partition</i></nobr></td>
+<td>Typically VFAT</td>
+<td>The Extended Boot Loader Partition (XBOOTLDR) used for the current boot is
+automatically mounted to <tt>/boot/</tt>, unless a different partition is
+mounted there (possibly via <tt>/etc/fstab</tt>) or the directory is non-empty
+on the root disk. This partition type is defined by the <a
+href="https://systemd.io/BOOT_LOADER_SPECIFICATION/">Boot Loader
+Specification</a>.</td> </tr>
+
+<tr>
+<td><nobr><tt>0fc63daf-8483-4772-8e79-3d69d8477de4</tt></nobr></td>
+<td><nobr><i>Other Data Partitions</i></nobr></td>
+<td>Any native, optionally in LUKS</td>
+<td>No automatic mounting takes place for other Linux data partitions. This
+partition type should be used for all partitions that carry Linux file
+systems. The installer needs to mount them explicitly via entries in
+<tt>/etc/fstab</tt>. Optionally, these partitions may be encrypted with LUKS.</td>
+</tr>
+
+</table>
+
+Other GPT type IDs might be used on Linux, for example to mark software RAID or
+LVM partitions. The definitions of those GPT types is outside of the scope of
+this specification.
+
+## Partition Names
+
+For partitions of the types listed above it is recommended to use
+human-friendly, descriptive partition names in the GPT partition table, for
+example "*Home*", "*Server* *Data*", "*Fedora* *Root*" and similar, possibly
+localized.
+
+## Partition Flags
+
+For the root, server data, home, variable data, temporary data and swap
+partitions, the partition flag bit 63 ("*no-auto*") may be used to turn off
+auto-discovery for the specific partition.  If set, the partition will not be
+automatically mounted or enabled.
+
+For the root, server data, home, variable data and temporary data partitions,
```

Comment:
/var can be read-only ? i thought it was one of the few places that needed to be R/W...

Reply:
it can#t be realistically. But I saw no reason to prohibit the flag. i.e. I think it's OK supporting that flag just to make things more systematic even if actually deploying things like that makes no sense.

But it might be useful for testing: making sure that if /var is read-only by accident the failure modes operate correctly.

Reply:
could this be mentionned ? i mean... for testing purpose it's simpler to just use fstab...

and out of the 5 partitions you mention, 3 make no sense to have RO (home, var/tmp and var) according to man:file-hierarchy. in particular /var/tmp would have weird results.

Reply:
will add